### PR TITLE
[10.0][IMP] Add variable qty to contract template view

### PR DESCRIPTION
**Pull request by lasley**
Allows templating of the variable qty lines by updating the `account.analytic.contract` view.

### CODE REVIEW

File: contract_variable_quantity/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {
     'name': 'Variable quantity in contract recurrent invoicing',
-    'version': '10.0.1.0.0',
+    'version': '10.0.1.0.1',
```
Comment (pedrobaeza) [inline contract_variable_quantity/__manifest__.py]: Should change minor version, not revision

**Review by pedrobaeza — APPROVED** [review]
Please change version

**Comment by lasley** [comment]
Thanks for the review @pedrobaeza - done

**Comment by rafaelbn** [comment]
Hi @lasley , sorry but I'm not able to create contract templates. I tried https://github.com/OCA/contract/issues/68 and also same error if I go to Accounting -> Configuration -> Contracts -> Contracts templates

**Comment by lasley** [comment]
@rafaelbn - looks like the issue is related to pricelists. For the purpose of this review, we can just not select a pricelist when creating a contract template in order to circumvent. 

I'll submit a separate PR for the fix in order to avoid expanding the scope of this feature with a bug fix.

**Review by rafaelbn — APPROVED** [review]
Thanks1

**Pull request merged**